pytorch_lightning/loggers/wandb.py

```diff
@@ -46,19 +46,20 @@ class WandbLogger(LightningLoggerBase):
 
     Args:
         name: Display name for the run.
-        save_dir: Path where data is saved.
+        save_dir: Path where data is saved (wandb dir by default).
         offline: Run offline (data can be streamed later to wandb servers).
         id: Sets the version, mainly used to resume a previous run.
+        version: Same as id.
         anonymous: Enables or explicitly disables anonymous logging.
-        version: Sets the version, mainly used to resume a previous run.
         project: The name of the project to which this run will belong.
         log_model: Save checkpoints in wandb dir to upload on W&B servers.
-        experiment: WandB experiment object.
         prefix: A string to put at the beginning of metric keys.
+        sync_step: Sync Trainer step with wandb step.
```

Comment:
Can you put this after the experiment flag, so that old args order is still working?

Reply:
I changed the order within the function itself.
For the description of args, I let this order because I thought it made more sense. For example `id` and `version` are the same thing. Is that ok?